### PR TITLE
[Feature] Add pacdeps to -Qi

### PR DESCRIPTION
**Pull request by Henryws**

### CODE REVIEW

File: misc/scripts/install-local.sh [hunk 1]
```diff
@@ -220,6 +222,9 @@ if [[ $removescript == "yes" ]] ; then
 fi
 echo "maintainer=\"$maintainer"\" | sudo tee -a /var/log/pacstall_installed/"$PACKAGE" >/dev/null
 echo "dependencies=\"$depends"\" | sudo tee -a /var/log/pacstall_installed/"$PACKAGE" >/dev/null
+if [[ -n $pacdepslist ]]; then
+    echo "pacdeps=\"$pacdepslist"\" | sudo tee -a /var/log/pacstall_installed/"$PACKAGE" >/dev/null
```
Comment (wizard-28) [inline misc/scripts/install-local.sh:226]:
```suggestion
    echo "pacdeps=\"${pacdepslist[@]}"\" | sudo tee -a /var/log/pacstall_installed/"$PACKAGE" >/dev/null
```

File: pacstall [hunk 2]
```diff
@@ -377,6 +377,9 @@ while [[ "$1" =~ ^- && ! "$1" == "--" ]]; do
       echo "description: ""$description"""
       echo "date installed: ""$date"""
       echo "maintainer: ""$maintainer"""
+      if [[ -n $pacdepslist ]]; then
+          echo "pacstall dependencies: ""$pacdepslist"""
```
Comment (wizard-28) [inline pacstall:381]:
```suggestion
          echo "pacstall dependencies: ""${pacdepslist[@]}"""
```

**Comment by wizard-28** [comment]
Please follow the tagging scheme from the next PR

**Comment by Henryws** [comment]
> Please follow the tagging scheme from the next PR

??

**Comment by wizard-28** [comment]
#31 added auto labeler, as you have reviewed the code, please follow the tagging scheme configured there

**Review by wizard-28 — CHANGES_REQUESTED** [review]
And replace `@` with `*` to fix the codefactor issue probably

**Comment by Henryws** [comment]
good to go?

**Review by wizard-28 — APPROVED** [review]
I will squash merge this, you don't merge it

**Pull request merged**